pkgs/development/python-modules/protobuf/default.nix

```diff
@@ -18,15 +18,6 @@ buildPythonPackage rec {
   nativeBuildInputs = [ google_apputils pyext ];
   buildInputs = [ protobuf ];
 
-  patches = optional isPy37
```

Comment:
We could use `optional (isPy37 && (lib.versionOlder protobuf.version "3.7.0"))`.